README.md

```diff
@@ -54,6 +54,7 @@ The current set of feature sources are the following:
 - [CPUID][cpuid] for x86 CPU details
 - [Intel Resource Director Technology][intel-rdt]
 - [Intel P-State driver][intel-pstate]
+- [Network][network]
```

Comment:
I tried to search for a generic network link but couldn't come with a suitable one. I will remove this link and add the SRIOV link in features.